### PR TITLE
typescript-promises - Lindsay Jin

### PR DESCRIPTION
**Pull request by lindsay-jin**
Screen Shot 2024-02-29 at 10 35 07 AM

### CODE REVIEW

File: typescript-promises/typescript-promises-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+- What are the three states a Promise can be in?
+  pending: initial state, neither fulfilled nor rejected.
+  fulfilled: meaning that the operation was completed successfully.
+  rejected: meaning that the operation failed.
+
+- How do you handle the fulfillment of a Promise?
+
+- How do you handle the rejection of a Promise?
+
```
Comment (shawnkost) [inline typescript-promises/typescript-promises-quiz-notes.md]: Missing the quiz questions! It's okay if you answer them incorrectly, we will go over in the Q&A, but you always need to submit at least something

**Review by shawnkost — CHANGES_REQUESTED** [review]
Missing some quiz questions

File: typescript-promises/typescript-promises-quiz-notes.md (hunk 1 above)
Comment (lindsay-jin) [reply typescript-promises/typescript-promises-quiz-notes.md]: Updated! Completely forgot after finishing the reading!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**